app/grandchallenge/challenges/forms.py

```diff
@@ -604,21 +574,51 @@ def clean_status(self):
         return status
 
 
-class ChallengeRequestBudgetUpdateForm(
-    ChallengeRequestBudgetFieldValidationMixin, forms.ModelForm
-):
+class ChallengeRequestBudgetUpdateForm(forms.ModelForm):
+    algorithm_selectable_gpu_type_choices = forms.MultipleChoiceField(
+        initial=get_default_gpu_type_choices(),
```

Comment:
Shouldn't this have the current choices pre-selected, rather than the default choices?